src/Shouldly/ShouldlyMessage.cs

```diff
@@ -240,20 +240,21 @@ public string CreateExpectedErrorMessage()
         private static string CreateActualVsExpectedMessage(IShouldlyAssertionContext context)
         {
             var codePart = context.CodePart;
-            string message = string.Format(@"
-    {0}
-        {1}
-    {2}
-        but was
-    {3}",
-                codePart, context.ShouldMethod.PascalToSpaced(), context.Expected.ToStringAwesomely(), context.Actual.ToStringAwesomely());
+            var message = string.Format(@"
+            {0}
+                {1}
+            {2}
+                but was
+            {3}",
+                codePart, context.ShouldMethod.PascalToSpaced(), context.Expected.ToStringAwesomely(),
+                context.ShouldMethod == "ShouldNotBe" ? string.Empty : context.Actual.ToStringAwesomely());
 
             if (DifferenceHighlighter.CanHighlightDifferences(context))
             {
                 message += string.Format(@"
-        difference
-    {0}",
-                DifferenceHighlighter.HighlightDifferences(context));
+                difference
```

Comment:
Morning guys, thanks for the feedback. Would it not be better if we used a StringBuilder instead to insert new lines and tabs where needed rather than relying on hardcoded white space?
Something along the lines of:

    var messageSb = new StringBuilder();
            messageSb.AppendLine(codePart);
            messageSb.AppendLine(string.Format("\t{0}", context.ShouldMethod.PascalToSpaced()));
            messageSb.AppendLine(context.Expected.ToStringAwesomely());
            messageSb.AppendLine("\tbut was");
            
            if (context.ShouldMethod != "ShouldNotBe")
            {
                messageSb.AppendLine(context.Actual.ToStringAwesomely());
            }

            if (DifferenceHighlighter.CanHighlightDifferences(context))
            {
                messageSb.AppendLine("\tdifference");
                messageSb.AppendLine(DifferenceHighlighter.HighlightDifferences(context));
            }

            return messageSb.ToString();